src/extensions/scratch3_thymio/index.js

```diff
@@ -390,12 +412,13 @@ class Thymio {
         speed = parseInt(Math.abs(speed), 10);
         speed = parseInt(clamp(speed, Thymio.VMIN * 10 / 32, Thymio.VMAX * 10 / 32), 10);
 
-        if (angle === 0) {
+        if (angle === 0) {//FIXME not working 
             const largs = Array();
```

Comment:
What's the bug ?

Reply:
it seems that the origin idea was "if angle was zero to give only the speed to wheels" but the strange calculation do not provide something that works